.travis/check-posix-app-pty

```diff
@@ -96,6 +96,8 @@ send "extaddr\r\n"
 expect "Done"
 send "ipaddr\r\n"
 expect "Done"
+send "coex\r\n"
```

Comment:
check the expected output?

Reply:
I have checked if the output is "Done" to make sure that the function "GetCoexMetrics()" is called.